cdk/domino_cdk/agent.py

```diff
@@ -124,4 +127,20 @@ def generate_install_config(
             }
         )
 
+    if install.registry_username:
+        agent_cfg["private_docker_registry"] = {
+            "server": "quay.io",
+            "username": install.registry_username,
+            "password": install.registry_password,
+        }
+
+    if install.gcr_credentials:
```

Comment:
I've seen a few people have started recommending the embedded charts over GCR, does that simplify this a bit?

Reply:
Yeah, that's a good idea. Takes the required creds down by one.

Reply:
I've nuked the gcr creds and am now just statically setting `{"version": 3, cache_path: "charts"}` . I wasn't more specific on the dir since the root is different between the deployer and the agent, but should be relative to the current working dir.